tensorflow/stream_executor/rocm/rocm_blas.cc

```diff
@@ -291,67 +268,6 @@ static string ToString(rocblas_status status) {
   }
 }
 
-// rocBLAS has interfaces that permit pointers to be passed from either the host
-// memory space or the device memory space; however, you must instruct it as to
```

Comment:
I think it would be worthwhile to keep a commit whether or not TF is using BLAS w/ the pointers in host or GPU memory.

Reply:
these old codes can still be traced back through commit history